README.md

```diff
@@ -1,3 +1,273 @@
+
+    ```bash
+    # setup for calling sow via the wrapper
+    git clone "https://github.com/gardener/sow"
+    cd sow
+    make # will build and tag the image
+    export PATH=$PATH:$PWD/docker/bin
+    ```
+
+    > The wrapper script starts `sow` in a Docker container, executes the sow command with the given arguments, and then exits and removes the container again. The script also mounts parts of your file system into that container to allow `sow` to persist the state in directory `landscape` which you will create in the next step.
+
+1. Create a directory for your Gardener landscape and clone this repository into a subdirectory called `crop`:
+
+    ```bash
+    mkdir landscape
+    cd landscape
+    git clone "https://github.com/gardener/garden-setup" crop
+    ```
+
+1. In your landscape folder, create a configuration file called `acre.yaml`. The structure of the configuration file is described [below](#configuration-file-acreyaml).
+
+    > Do not use file `acre.yaml` in folder `crop`. This file is used internally by the installation tool.
+
+1. If you created the base cluster using GKE you need to convert your `KUBECONFIG` to one that uses basic authentication with Google-specific configuration parameters:
+
+    1. Set your `KUBECONFIG` environment variable to a file that does not exist. Do *not* use the one specified in your acre.yaml file.
+
+    1. Download the GKE kubeconfig:
+
+        ```bash
+        gcloud container clusters get-credentials <your_cluster> --zone <your_zone> --project <your_project>
+        ```
+
+    1. Call `crop/bin/convertkubeconfig 3.1`. When asked for credentials, enter the ones that the GKE dashboard shows when clicking on `show credentials`.
+
+    This will create a kubeconfig that uses basic auth. It will be created at the path specified in your `acre.yaml` file.
+
+1. Open a second terminal window and enter the following command to watch the progress of the Gardener installation:
+
+    ```bash
+    watch -d kubectl -n garden get pods,ingress,sts,svc
+    ```
+
+1. In your first terminal window, use the following command to deploy Gardener on your base cluster:
+
+    ```bash
+    sow deploy -A
+    ```
+
+`sow` now starts to install Gardener in your base cluster. The installation can take about 30 minutes. `sow` prints out status messages to the terminal window so that you can check the state of the installation. The other terminal window will show the newly created Kubernetes resources after a while and if their deployment was successful (their status should be `Running`). Finally, the Gardener dashboard is deployed. The last status message before `sow` exits is something like this:
+```bash
+===================================================================
+Dashboard URL -> https://gardener.ingress.<value of `landscape.cluster.domain`>
+===================================================================
+generating exports
+*** species dashboard deployed
+```  
+
```

Comment:
Maybe we should change this. The dashboard is not necessarily the last component that is deployed, so this might be confusing. 

Furthermore, it is now possible to also get the dashboard url by calling `sow url` (or reading the `dashboard_url` file in the dashboard's export folder).

Reply:
done